cmd/manager/main.go

```diff
@@ -70,7 +70,8 @@ func main() {
 
 	// Initialize cluster actuator.
 	clusterActuator := cluster.NewActuator(cluster.ActuatorParams{
-		Client: cs.ClusterV1alpha1(),
+		Client:         cs.ClusterV1alpha1(),
+		LoggingContext: "[cluster actuator]",
```

Comment:
We'll have to keep the klog stuff like `flag.Parse` alive in the manager until we have a better solution